extra/1-currency-conversion.js

```diff
@@ -15,7 +17,13 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+const convertToBRL = (amount) => {
+  let deductionFee = amount / 100;
+  let poundToConvert = amount - deductionFee;
+  let payOut = +(poundToConvert * 5.7).toFixed(2);
```

Comment:
toFixed converts a number into a string. rather than converting it to string, and then back to number with the plus sign, we can use Math.floor(x * 100) / 100 to round the number to 2 decimal places.